.circleci/config.yml

```diff
@@ -330,6 +335,7 @@ workflows:
             - lint
             - pkg-kubelet
             - pkg-cri-containerd
+            - pkg-kube-e2e-test
```

Comment:
You are missing a `linuxkit pkg push --nobuild` for this new package in the `push-pkgs-to-hub` step.

Reply:
Well spotted!